### ISSUE TITLE
Webm Support

### ISSUE THREAD

**Issue by Sora012** [aler9/rtsp-simple-server#1427]
## Describe the feature
VP8/VP9 Playback support

I'm requesting WebM support (VP8/VP9) to my understanding there are not many hardware encoders available, however it has a wide range of supported devices and browsers from my understanding. Its also capable of being live streamed.

**Comment by aler9** [comment]
Hello, VP8 and VP9 are fully supported, check the compatibility matrix in the README.

**Comment by Sora012** [comment]
Maybe this issue wasn't clear but VP8 & VP9 don't appear to work for output/playback other than WebRTC/RTSP Clients. I was requesting that they be made as available formats similiar to the HLS output (served by web). I spent a long while trying to figure out how to make it available via FFMPEG but wasn't able to, I don't know enough about FFMPEG to convert the RTSP stream into WebM (Live) and all of my attempts I had constant buffering and audio/video desync issues.

WebRTC/RTSP/MP4/AAC/DASH/HLS are all unsupported in the game I am playing and can only really use OGG,WebM formats. Which is the sole reason for this request for WebM playback in a style similiar to how HLS is served.

**Comment by aler9** [comment]
HLS can be extended to support VP8/VP9, since there's a specification that describes how to incapsulate VP8 frames into MP4 files:

https://www.webmproject.org/vp9/mp4/

The problem is that no browser supports it.

Another option consists into hacking HLS and replacing MP4 files with WEBM files, but again, no browser supports it. If there's no client support there's no reason to develop server side support for anything.

Just re-encode, or even better, find a way to use WebRTC.

**Comment by Sora012** [comment]
This project [stream-m](https://github.com/vbence/stream-m) is able to serve a live WebM file over HTTP that is ingested via RTMP and is compatiable with <video> tag. The issue with this project is it lacks external authentication and also appears to not be actively developed.

I don't know the magic behind stream-m, but thats the idea i'm trying to suggest atleast. Last I tried that project for testing purposes, it worked fine. However it would be annoying to modify a config and restart it each time I want to allow someone to stream or to deny a stream by restarting knocking everyone else off.

As for using WebRTC or anything else, I can't access or modify the game. Its protected by anti-cheat, and even if I did manage to do so without being banned no one else would be able to see videos/music due to lack of codec support and it would be wasted effort.
